tests/unit/payment-methods/test-class-upe-payment-gateway.php

```diff
@@ -392,25 +404,32 @@ public function test_process_redirect_payment_intent_processing() {
 		)[0];
 
 		$this->assertContains( 'authorized', $note->content );
-		$this->assertEquals( $result_order->get_meta( '_intent_id', true ), $intent_id );
-		$this->assertEquals( $result_order->get_meta( '_charge_id', true ), $charge_id );
-		$this->assertEquals( $result_order->get_meta( '_intention_status', true ), $intent_status );
-		$this->assertEquals( $result_order->get_meta( '_payment_method_id', true ), $payment_method_id );
-		$this->assertEquals( $result_order->get_meta( '_stripe_customer_id', true ), $customer_id );
-		$this->assertEquals( $result_order->get_status(), 'on-hold' );
+		$this->assertEquals( $intent_id, $result_order->get_meta( '_intent_id', true ) );
+		$this->assertEquals( $charge_id, $result_order->get_meta( '_charge_id', true ) );
+		$this->assertEquals( $intent_status, $result_order->get_meta( '_intention_status', true ) );
+		$this->assertEquals( $payment_method_id, $result_order->get_meta( '_payment_method_id', true ) );
+		$this->assertEquals( $customer_id, $result_order->get_meta( '_stripe_customer_id', true ) );
+		$this->assertEquals( 'on-hold', $result_order->get_status() );
```

Comment:
Not really related to the changes in this PR again, but these parameters were the wrong way around and it was confusing me (also, I'm pretty sure I was the one who put them in the wrong order in the first place).